src/database/initialization/user_to_json.sql

```diff
@@ -16,6 +16,9 @@ BEGIN
       SELECT user_changemaker_permissions.changemaker_id, jsonb_agg(user_changemaker_permissions.permission) AS permissions
       FROM user_changemaker_permissions
       WHERE user_changemaker_permissions.user_keycloak_user_id = "user".keycloak_user_id
+        AND (user_changemaker_permissions.not_after IS NULL
+          OR user_changemaker_permissions.not_after > now()
+        )
```

Comment:
Slight style inconsistency.

Reply:
Fixed this by adding an `is_expired()` function!